docs/administration/configuration/configuring-access/index.md

```diff
@@ -0,0 +1,6 @@
+# Configuring user access
+
+By default confirmation of a user’s email address on sign up is enabled.
+The user is unable to sign in until they confirm their email address.
```

Comment:
With the [latest changes](https://github.com/reanahub/reana-server/pull/332/commits/f0da75b502e47d801654c9da07430755db847416), admins can opt-out of this feature. Might be a good idea to point out how to do it in this section.

A rough attempt:

> To disable the confirmation of user's email address, add `REANA_USER_CONFIRMATION: false` Helm value  to `components.reana_server.environment`.

Reply:
Here is what it looks like now:
![Screenshot from 2021-01-11 10-09-03](https://user-images.githubusercontent.com/4990025/104162549-1cc99d00-53f5-11eb-8891-ab69c2462ca5.png)